.env

```diff
@@ -1,7 +1,18 @@
 # Twilio API credentials
-export TWILIO_ACCOUNT_SID=AC807f0beda34c3b254e0cf785a42420af
-export TWILIO_AUTH_TOKEN=4de3c67a78763ae5c4e67a2cadce45f9
+export TWILIO_ACCOUNT_SID=AC1ead505ca3ff85324c9c26dbceed3296
```

Comment:
Can you please remove this and every other secret here and use an environment variable instead. Thanks.